action.yml

```
@@ -6,36 +6,55 @@ inputs:
     default: "--verbose --no-progress './**/*.md' './**/*.html'"
     required: false
   format:
-    description: "summary output format (e.g. json)"
+    description: "Summary output format (e.g. json)"
     default: "markdown"
     required: false
   output:
-    description: "summary output file path"
-    default: "lychee/out.md"
+    description: "Summary output file path"
+    default: "summary.md"
     required: false
   fail:
-    description: "fail entire pipeline on error (i.e. when lychee exit code is not 0)"
+    description: "Fail entire pipeline on error (i.e. when lychee exit code is not 0)"
     default: false
     required: false
   jobSummary:
-    description: "write Github job summary at the end of the job (written on Markdown output only)"
+    description: "Write Github job summary at the end of the job (written on Markdown output only)"
     default: true
     required: false
   lycheeVersion:
-    description: "use custom version of lychee link checker"
+    description: "Use custom version of lychee link checker"
     default: 0.10.0
     required: false
+  createIssue:
+    description: "Create Github issue if link check fails"
+    default: true
+    required: false
+  issueTitle:
+    description: "Title of Github issue which will be created if link check fails"
+    default: Link Checker Report
```

May be better to scope the issue with a prefix that indicates in the title it's a CI issue, and more specific hint of the content (_since while it does contain a report, it's only being reported when failures are caught?_):

```suggestion
    default: 'ci(lint): Some monitored links are failing'
```